.github/resources/payload-notification.json

```diff
@@ -1,13 +1,31 @@
 {
```

Comment:
Some of the fields in this payload are not showing up correctly. Attached is a screenshot of the GitHub CI run from https://github.com/slackapi/slack-github-action/pull/163 (which is a copy of this PR), and how it looks like in our Slack workspace where the message gets posted:

<img width="521" alt="Screenshot 2023-02-28 at 3 32 26 PM" src="https://user-images.githubusercontent.com/52645/221972226-23fc149d-fee9-4225-a059-1b1cedb160f5.png">

In particular, it seems the following fields do not work as desired from the `integration_test_file_payload` test from `main.yml`: `env.JOB_STATUS`, `github.payload.head_commit.url`, `github.payload.head_commit.author.name`, `github.payload.head_commit.message`.

Any idea why this might be happening?

Reply:
same seems to happen for us